qmlui/qlcplus_fr_FR.ts

```diff
@@ -2873,7 +2878,7 @@ Niveau d&apos;accès&#xa0;:</translation>
     <message>
         <location filename="qml/fixturesfunctions/3DView/SettingsView3D.qml" line="614"/>
         <source>Select a mesh file</source>
-        <translation type="unfinished"></translation>
+        <translation>Sélectionner un fichier 3D</translation>
```

Comment:
Why not "sélectionner un fichier mesh" ?

Reply:
The word "mesh" does not mean anything in French, and it is a file which contains a 3D object.

Reply:
What about "Sélectionner un fichier de maille" or "Sélectionner un fichier de maillage" ? (if "maille/maillage" translates righly to "mesh" obvioulsy). The reasoning behind that is to avoid using 3D while it's not there in the original English version (it's not "Select a 3D file").

Reply:
Justement, le travail du traducteur est d'adapter les textes afin qu'ils soient **compréhensibles** par une personne parlant la langue.

De plus, dans le contexte de cette fenêtre de sélection de fichiers, il est précisé que ce sont des fichiers 3D qui sont attendus :
```
FileDialog
                {
                    id: meshDialog
                    visible: false
                    title: qsTr("Select a mesh file")
                    folder: View3D.meshDirectory
                    nameFilters: [ qsTr("3D files") + " (*.obj *.dae *.3ds *.py *.stl *.blend)", qsTr("All files") + " (*)" ]

                    onAccepted: View3D.createGenericItem(fileUrl, -1)
                }
```

Demandez vous si vous comprenez mieux l'objectif d'un bouton "Sélectionner un fichier de maillage" ou "Sélectionner un fichier 3D".

Reply:
https://github.com/mcallegari/qlcplus/blob/master/qmlui/qml/fixturesfunctions/3DView/SettingsView3D.qml#L614

Reply:
S'il est plus intéressant d'utiliser le vocable "3D", alors il faut le faire remonter à la traduction d'origine. Le travail du traducteur est premièrement de respecter le travail de l'auteur, sinon c'est la foire au "moi je pense que c'est mieux", et en cas de travail à plusieurs (comme c'est le cas ici), les batailles d'égo ne sont pas loin.

@mcallegari : we're discussing here an issue. Would you think it could make sense to change the string "Select a mesh file" to "Select a 3D file" ? Or would that change the original meaning?

Reply:
To make it clear, I am annoyed by your comment. That's why I answered in French, to be sure to say exactly what I wanted. That said, I understand what you say about ego conflicts, they are dangerous in communication between people.

To come back to the question, I'll say that the denomination "mesh file" in the English text should not force the French translation to be "fichier de maillage" because the denomination "fichier de maillage" is not the clearest way to say what it is in French, even if [SolidWorks](http://help.solidworks.com/2018/french/SolidWorks/sldworks/t_importing_mesh_file.htm) uses it. If we were talking to engineers or 3D modelers, it would have made sense. But we are talking to light technicians, all the end user care is what is in the file, not the way it is represented.

Anyway, that's my point of view. Do what you want with it.

Reply:
PS: All of that to say I don't think I'm changing the meaning to me it's more clear what a "fichier 3D" is than a "fichier de maillage". Maybe we could translate it to "fichier de maillage 3D" ?

Reply:
Hey guys, sorry for the delay.
To be honest I haven't checked how other softwares call "meshes" or "3D" files.
I just used "mesh" because the folder where the bundled ones are stored is called "Meshes" and because Qt calls them that way.
To me it doesn't make much difference. If you believe French people will be less confused with "3D files", then please use that.
Otherwise, see this: http://www.assimp.org/
Maybe "Select a 3D model file" can make everyone happy?

Reply:
>  Maybe "Select a 3D model file" can make everyone happy?

:+1: